infra/gcp-broad/gcp-ar-cleanup-policy.txt

```diff
@@ -114,6 +114,7 @@
                 "create_certs_image",
                 "curl",
                 "git-make-bash",
+                "gpu",
```

Comment:
Reminder for me when this merges to update the policy with `gcloud`.

Reply:
@jigold can you PR a change to add a build step that applies the gap ar cleanup policy on deploys?